book/src/intro.md

```diff
@@ -15,46 +18,53 @@ _Documentation for the Kona project._
 
 ## Introduction
 
-Kona is _the_ monorepo for the Rust Optimism <a href="https://specs.optimism.io/">OP Stack</a>.
+Kona is _the_ monorepo for the Rust <a href="https://specs.optimism.io/">OP Stack</a>.
 
-Kona provides a suite of libraries and build pipelines for developing verifiable Rust programs targeting
-{{#template ../templates/glossary-link.md root=./ ref=fault-proof-vm text=Fault Proof VMs}}.
+Kona provides a rich suite of libraries and build pipelines. Proof libraries are available for
+developing verifiable Rust programs targeting {{#template ../templates/glossary-link.md root=./ ref=fault-proof-vm text=Fault Proof VMs}}.
+These libraries provide tooling and abstractions around low-level syscalls, memory management,
+and other common structures that authors of verifiable programs will need to interact with.
+It also provides build pipelines for compiling `no_std` Rust programs to a format that can be
+executed by supported Fault Proof VM targets.
 
-It is built and maintained by members of [OP Labs][op-labs] as well as open source contributors,
-and is licensed under the MIT License.
+Protocol libraries are `no_std` compatible for use within the Fault Proof. These types are
```

Comment:
Nit: need to stop talking about fault proofs as "the proof," zk is here + the libs are unopinionated on the prover platform.